charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.5.5
 kubeVersion: ">=1.22.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.16.7
+version: 5.16.9
```

Comment:
@rakhbari Can you please rebase this? Argo CD changes should not be part of this PR.

Reply:
@pdrastil: The changes to `argo-cd` and `argo-workflows` charts came in when I was trying to resolved the DCO `sign-off` issue, which instructed me to do `git rebase HEAD~8 --signoff`. But now those commits have been pushed to my fork and therefore part of this PR. I'm pretty sure a `rebase` isn't going to solve it. I think I'm going to have to do `git checkout upstream/master --` on my fork for both `argo-cd/` and `argo-workflows/` folders, then re-commit to remove them from this PR. Do you have a better set of steps to remove those commits from this PR?

Reply:
Sorry to jump in like that :) I think this is a good plan.

P.S. I'm also waiting for the PR 🚀

Reply:
What I do in such case is to sync my main with upstream repo and then rebase the PR branch on top.